LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/WebSocketWriterRegistry.cs

```diff
@@ -177,7 +177,7 @@ async Task LoopAsync()
 
                     var prunedKeys = Prune();
 
-                    this.logger?.LogDebug($"Pruned closed WebSocket connections: {string.Join(",", prunedKeys)}");
+                    this.logger?.LogDebug("Pruned closed WebSocket connections: {PrunedKeys}", string.Join(",", prunedKeys));
```

Comment:
What's the reason for this change? Do we need structured logging here?

Reply:
Yes. A new code warning disallows log message templates that change with each invocation, i.e. forces us to use structured logging.

Reply:
Reverted this with a29c611156a5b208419fac20e931f8f8661cce72. I suppressed the warning for now, since the merge of #842 generated many more of these instances. We will address this as part of a separate task (linked in the description). Since I already fixed some of these warnings in an older version of the branch, I will not revert those.